lib/developer_portal/app/controllers/developer_portal/admin/account/personal_details_controller.rb

```diff
@@ -46,7 +46,7 @@ def deny_unless_can_update
   def user_params
     params.require(:user).permit([:current_password] +
                                    resource.special_fields +
-                                   resource.defined_fields.map(&:name))
+                                   resource.defined_fields.reject(&:read_only).map(&:name))
```

Comment:
Perhaps add a method instead of implementing the logic here. e.g.
```suggestion
                                   resource.writable_defined_fields.map(&:name))
```